boards/arm/96b_carbon/Kconfig.defconfig

```diff
@@ -10,4 +10,47 @@ if BOARD_96B_CARBON
 config BOARD
 	default 96b_carbon
 
+if I2C
+
+config I2C_1
+	def_bool y
+
+config I2C_2
+	def_bool n
+
+config I2C_3
+	def_bool n
+
```

Comment:
sure no problem; it seems that frdm_k64f was the wrong template to follow (since it was written by maintainers I thought it made sense, obviously not)